.github/workflows/ci.yml

```
@@ -22,21 +22,11 @@ jobs:
         run: bun install
         
       # Code quality checks
-      - name: Lint code
-        run: bun run lint
+      - name: Lint code with Biome
+        run: bunx biome lint --apply-unsafe ./src
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)